src/Robo/Plugin/Commands/ThemeWatchCommand.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Fire\Robo\Plugin\Commands;
+
+use Robo\Symfony\ConsoleIO;
+use Fire\Robo\Plugin\Commands\FireCommandBase;
+use Robo\Robo;
+
+/**
+ * Provides a command to watch the custom theme.
+ */
+class ThemeWatchCommand extends FireCommandBase {
+
+  /**
+   * Builds Projects theme.
+   *
+   * Usage Example: fire theme-watch
+   *
+   * @command local:theme:watch
+   * @aliases theme-watch, watch-theme, tw
+   *
+   */
+  public function themeWatch(ConsoleIO $io) {
+    $env = Robo::config()->get('local_environment');
+    $root = $this->getThemePath();
+    $tasks = $this->collectionBuilder($io);
+    $npmCommand = Robo::config()->get('local_theme_watch_script') ?: 'watch';
```

Comment:
1 - Please add this to the  fire.yml template
2- add a option for this into the new Fire init Wizard
3 - Setup the default value into the wizard and template, not over here

Reply:
Everything is done!